### ISSUE TITLE
Hooking other debuggers

### ISSUE THREAD

**Issue by dpo** [shuky19/sublime_debugger#22]
This is a general question, not an issue with your package. In my opinion, the Python world is seriously lacking proper integration between their debuggers and their text editors, and your sublime_debugger is exactly what I had in mind. A similar package for GDB is [SublimeGDB](https://github.com/quarnster/SublimeGDB) and there is [SublimeLLDB](https://github.com/filcab/SublimeLLDB) for LLDB.

I'm not an expert on debuggers but I do need to use them. My question is: how difficult would it be to extract a "generic" debugger interface from sublime_debugger (or any of the others), so users could write simple plugin interfaces to their favorite debugger? My own interest is in Python, but I imagine you could plug in a debugger for any language.

In other words, how specific is sublime_debugger to the Ruby debugger?

Thanks!
